yarn/pom.xml

```diff
@@ -109,6 +109,16 @@
       <properties>
         <jersey.version>1.9</jersey.version>
       </properties>
+      <!-- Enable hadoop-2.2 profile when the Hadoop version is 2.2.0, which is now 
+      the default. Those building against 2.2.x or some distro-specific variation of 
+      that string would still have to manually enable -Phadoop-2.2.
+      -->
+      <activation>
+       <property>
+        <name>hadoop.version</name>
```

Comment:
That's what the comment addresses.

The issue here is that if you just run "mvn -Pyarn test", the YARN tests will fail. So we want this profile to be activated to avoid that.

I guess one solution is to use `<activeByDefault>true</activeByDefault>`. My peeve with that approach is that the name is incredibly misleading - the above command line would fail the same way, because this property just activates the profile if no other profile has been manually activated.